### PR TITLE
PS sku changes: LoadBalancer, PublicIpAddress

### PR DESCRIPTION
**Pull request by EvgenyAgafonchikov**
## Description

---

This checklist is used to make sure that common guidelines for a pull request are followed. You can find a more complete discussion of PowerShell cmdlet best practices [here](https://msdn.microsoft.com/en-us/library/dd878270(v=vs.85).aspx).

- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md).**
- [x] **If changes were made to any cmdlet, the XML help was regenerated using the [platyPSHelp module](https://github.com/Azure/azure-powershell/blob/preview/documentation/help-generation.md).**
- [x] **If any large changes are made to a service, they are reflected in the respective [change log](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#updating-the-change-log).**

### [General Guidelines](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/preview/documentation/cleaning-up-commits.md).
- [x] The pull request does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/preview/documentation/breaking-changes/breaking-changes-definition.md) (unless a major version change occurs in the assembly and module).

### [Testing Guidelines](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#testing-guidelines)
- [x] Pull request includes test coverage for the included changes.
- [x] PowerShell scripts used in tests should do any necessary setup as part of the test or suite setup, and should not use hard-coded values for locations or existing resources.

### [Cmdlet Signature Guidelines](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#cmdlet-signature-guidelines)
- [x] New cmdlets that make changes or have side effects should implement `ShouldProcess` and have `SupportShouldProcess=true` specified in the cmdlet attribute. You can find more information on `ShouldProcess` [here](https://github.com/Azure/azure-powershell/wiki/PowerShell-Cmdlet-Design-Guidelines#supportsshouldprocess).
- [x] Cmdlet specifies `OutputType` attribute if any output is produced - if the cmdlet produces no output, it should implement a `PassThru` parameter.

### [Cmdlet Parameter Guidelines](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#cmdlet-parameter-guidelines)
- [x] Parameter types should not expose types from the management library - complex parameter types should be defined in the module.
- [x] Complex parameter types are discouraged - a parameter type should be simple types as often as possible. If complex types are used, they should be shallow and easily creatable from a constructor or another cmdlet.
- [x] Cmdlet parameter sets should be mutually exclusive - each parameter set must have at least one mandatory parameter not in other parameter sets.

### CODE REVIEW

**Comment by DeepakRajendranMsft** [comment]
@cormacpayne please take a look at this when you are free

File: src/ResourceManager/Network/Commands.Network/Models/PSResourceSku.cs [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+namespace Microsoft.Azure.Commands.Network.Models
+{
+    public class PSResourceId
```
Comment (cormacpayne) [inline src/ResourceManager/Network/Commands.Network/Models/PSResourceSku.cs]: @EvgenyAgafonchikov file name is `PSResourceSku` but class name is `PSResourceId`

File: src/ResourceManager/Network/Commands.Network/help/Add-AzureRmLoadBalancerRuleConfig.md [hunk 2]
```diff
@@ -90,6 +92,21 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -DisableOutboundSNAT
+DisableOutboundSNAT
```
Comment (cormacpayne) [inline src/ResourceManager/Network/Commands.Network/help/Add-AzureRmLoadBalancerRuleConfig.md]: @EvgenyAgafonchikov would you mind updating this description to be more meaningful to the user?

File: src/ResourceManager/Network/Commands.Network/help/New-AzureRmLoadBalancerRuleConfig.md [hunk 3]
```diff
@@ -96,6 +97,21 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -DisableOutboundSNAT
+DisableOutboundSNAT
```
Comment (cormacpayne) [inline src/ResourceManager/Network/Commands.Network/help/New-AzureRmLoadBalancerRuleConfig.md]: @EvgenyAgafonchikov same comment

File: src/ResourceManager/Network/Commands.Network/help/Set-AzureRmLoadBalancerRuleConfig.md [hunk 4]
```diff
@@ -94,6 +96,21 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -DisableOutboundSNAT
+DisableOutboundSNAT
```
Comment (cormacpayne) [inline src/ResourceManager/Network/Commands.Network/help/Set-AzureRmLoadBalancerRuleConfig.md]: @EvgenyAgafonchikov same comment

File: src/ResourceManager/Network/Commands.Network/help/Add-AzureRmLoadBalancerRuleConfig.md (hunk 2 above)
Comment (EvgenyAgafonchikov) [reply src/ResourceManager/Network/Commands.Network/help/Add-AzureRmLoadBalancerRuleConfig.md]: @DeepakRajendranMsft could you please contact owners of this update to ask them are they good with renaming of the parameter and what name would be applicable there from their point of view?

File: src/ResourceManager/Network/Commands.Network/Models/PSResourceSku.cs (hunk 1 above)
Comment (EvgenyAgafonchikov) [reply src/ResourceManager/Network/Commands.Network/Models/PSResourceSku.cs]: Dropped this file as unused

File: src/ResourceManager/Network/Commands.Network/help/Add-AzureRmLoadBalancerRuleConfig.md (hunk 2 above)
Comment (EvgenyAgafonchikov) [reply src/ResourceManager/Network/Commands.Network/help/Add-AzureRmLoadBalancerRuleConfig.md]: Replaced description with one from rest api spes

**Comment by DeepakRajendranMsft** [comment]
@Cormacm looks like @EvgenyAgafonchikov  has addressed the comments. please take a another look. thanks!

**Pull request merged**